src/System.Net.Sockets/tests/FunctionalTests/SendReceive.cs

```diff
@@ -125,6 +125,77 @@ private static void SendToRecvFromAsync_Datagram_UDP(IPAddress leftAddress, IPAd
             }
         }
 
+        private static void SendToRecvFromAsync_UdpClient_Datagram_UDP(IPAddress leftAddress, IPAddress rightAddress)
+        {
```

Comment:
nit: Why use capital `UDP` in the test name but other parts of the test name use `Udp` ?